zScanner/Coordinators/AppCoordinator.swift

```diff
@@ -89,7 +94,6 @@ extension AppCoordinator: LoginFlowDelegate {
 extension AppCoordinator: DocumentsFlowDelegate {
     func logout() {
         removeUserSession()
-        SeaCatClient.reset()
```

Comment:
@ateska 
How does the login work now? I don't see any call to `/logout` endpoint

Reply:
SeaCat identity is no longer tightly connected with a logged user. So SeaCat reset is not truly needed here ... or maybe yes, but that will be seen later.